beefOnion.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Noodles!</title>
+</head>
+<body>
+  <h1>Chicken Noodle Soup</h1>
+  <img src="imgs/beef-onion-noodles.jpg" alt="beef-onion-noodles-img" width="700" height="400" >
+  <h2>Ingredients</h2>
+    <p>Makes 4 servings</p>
+    <ul>
+      <li>1 pound of beef</li>
+      <li>1 tablespoon of cornstarch</li>
+      <li>1/3 cups of soy sauce</li>
+      <li>1 1/2 tablespoon of dark soy sauce</li>
+      <li>1 tablespoon of granulated sugar</li>
+      <li>1 small onion</li>
+      <li>16 oz of spaghetti</li>
+    </ul>
+  <h2>Intructions</h2>
+  <ol>
+    <li>Follow the directions of the Spaghetti (16 oz) package for al dente pasta, but subtract 1-2 minutes from what they
+    suggest. We will be cooking this again in a frying pan so we want to under-cook the pasta.</li>
+    <li>Once the noodles are done, pour in into a strainer and lightly rinse off the extra starch and set aside.</li>
+    <li>Cut the Onion (1) in half, then and slice them into thin half moon slices and set aside.</li>
+    <li>Combine the Dark Soy Sauce (1 1/2 Tbsp) , Soy Sauce (1/3 cup) , and Granulated Sugar (1 Tbsp) into 1 bowl and set aside.</li>
+    <li>Cut the Beef (1.1 lb) thinly against the grain and put it into a bowl.</li>
+    <li>Marinate the beef with 2-3 teaspoons of the soy sauce mixture from step 2 and Corn Starch (1 tsp) for 5-10 minutes.</li>
+    <li>Add in the onions and cook it for 3 minutes until soft and golden.</li>
+    <li>Add in the beef and cook for 3-5 minutes until it looks done.</li>
+    <li>Add in the spaghettini and then add the sauce mixture on top of everything and cook for 2-3 minutes until everything is
+    nice and brown and evenly mixed. Serve and enjoy!</li>
+  </ol>
+  <p>Total prep time : 15 minutes</p>
+  <button onclick="document.location='index.html'"> Back To Noodles Recipes</button>
```

Comment:
interesting use of `onclick` attribute here. usually we would use an `a` tag instead as it helps with accessibility